### PR TITLE
Upgrade to elm 0.19

### PR DESCRIPTION
**Pull request by EverybodyKurts**
This merge request is to upgrade the `elm-all-set` to be compatible with `elm 0.19`.

There is a major breaking change with this upgrade: I have removed the `eeue56/elm-all-dict` dependency in favor of `pzp1997/assoc-list`. I did this because `eeue56/elm-all-dict` hasn't been update to be compatible with `0.19` yet and relied on some js native functionality.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,14 +34,10 @@ install:
...
   - npm --version
-  - npm install -g elm@0.18.0 elm-test@0.18.7 elm-verify-examples@1.0.2 elm-format@exp
-  - mv $(npm config get prefix)/bin/elm-make $(npm config get prefix)/bin/elm-make-old
-  - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-make-old "$@"' > $(npm config get prefix)/bin/elm-make
-  - chmod +x $(npm config get prefix)/bin/elm-make
-  - travis_retry elm-package install --yes
-  - cd tests
-  - travis_retry elm-package install --yes
-  - cd ..
+  - npm install -g elm@0.19.0 elm-test@0.19.0-rev4 elm-verify-examples@3.0.1 elm-format@elm0.19.0
```
Comment (amitaibu) [inline .travis.yml]: Note this integration -- https://blog.travis-ci.com/2018-11-20-elm-is-now-supported-on-travis-ci

File: src/EverySet.elm [hunk 2]
```diff
@@ -55,14 +42,14 @@ based on [EveryDict](http://package.elm-lang.org/packages/eeue56/elm-all-dict/la
 
 -}
 
-import EveryDict exposing (EveryDict)
+import AssocList as EveryDict exposing (Dict)
```
Comment (amitaibu) [inline src/EverySet.elm]: I think we can get rid of the word `EveryDict` here and `import AssocList exposing (Dict)`

**Review by amitaibu — COMMENTED** [review]
@KurtRMueller Happy to see you working on this, thanks 👍

File: .travis.yml (hunk 1 above)
Comment (EverybodyKurts) [reply .travis.yml]: @amitaibu Thank you. This greatly simplified the ci config!

**Review by EverybodyKurts — COMMENTED** [review]
@amitaibu I will change the references to `EveryDict`.

**Comment by amitaibu** [comment]
Awesome work @KurtRMueller , thanks for that!

I have given have commit access, so now you are a co-maintainer -- I hope that is ok with you. If so, feel free to merge, and publish :)

**Comment by EverybodyKurts** [comment]
Thanks for that @amitaibu. I think I can handle that, lol. Any sort of code changes I write I'll still create a merge request for :).

**Comment by amitaibu** [comment]
@KurtRMueller great! Can you please publish to elm-package as-well?

**Comment by EverybodyKurts** [comment]
Done!

**Pull request merged**